js/src/common/utils/BasicEditorDriver.ts

```diff
@@ -32,19 +34,25 @@ export default class BasicEditorDriver implements EditorDriverInterface {
     this.el.onclick = callInputListeners;
     this.el.onkeyup = callInputListeners;
 
-    this.el.addEventListener('keydown', function (e) {
-      if ((e.metaKey || e.ctrlKey) && e.key === 'Enter') {
-        params.onsubmit();
-      }
+    this.el.addEventListener('keydown', (e) => {
+      this.keyHandlers(params)
+        .toArray()
+        .forEach((handler) => handler(e));
     });
 
     dom.append(this.el);
   }
 
-  protected setValue(value: string) {
-    $(this.el).val(value).trigger('input');
+  keyHandlers(params): ItemList {
+    const items = new ItemList();
 
-    this.el.dispatchEvent(new CustomEvent('input', { bubbles: true, cancelable: true }));
+    items.add('submit', function (e) {
+      if ((e.metaKey || e.ctrlKey) && e.key === 'Enter') {
```

Comment:
Is Win + Enter causing a post to submit intentional, as it seems like it will do.

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/metaKey

Reply:
In Markdown, we check against the user agent to decide between Cmd and Ctrl.

Maybe we could add the getOs helper I had in a PR a while back to make this info part of core and available to extensions instead of duplicating logic.

Something like `app.operatingSystem` or `window.__operatingSystem`?

Reply:
In the long term, I'd to introduce some form of "normalize keys" util like exists in prosemirror, which turns `Meta-KEY` into the appropriate cmd vs ctrl.

For now, I'm not sure this PR is the best place to change existing behavior?

Reply:
If this is the current action of MDArea/Prosmirror then I say we keep it this way for now and modify it later once we normalize things.